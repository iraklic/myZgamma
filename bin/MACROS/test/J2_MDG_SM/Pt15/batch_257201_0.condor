000 (257201.000.000) 10/28 11:59:45 Job submitted from host: <131.225.191.162:59609>
...
001 (257201.000.000) 10/28 12:00:06 Job executing on host: <131.225.206.216:56971>
...
005 (257201.000.000) 10/28 12:00:10 Job terminated.
	(1) Normal termination (return value 1)
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	6427  -  Run Bytes Sent By Job
	37404016  -  Run Bytes Received By Job
	6427  -  Total Bytes Sent By Job
	37404016  -  Total Bytes Received By Job
...
