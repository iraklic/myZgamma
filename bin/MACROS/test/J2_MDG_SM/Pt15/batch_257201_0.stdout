
CMSSW on Condor

started at Fri Oct 28 12:00:07 CDT 2011

parameter set:
CONDOR_CLUSTER: 257201
CONDOR_PROCESS: 0
CMSSW_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_4_2_0
RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam
PARAMETER_SET: output_Mu_B_S.txt
NUM_EVENTS_PER_JOB: 

Writing final parameter-set: batch_257201_0.py to RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam

run: myMinexam > batch_257201_0.log 2>&1\n

finished at Fri Oct 28 12:00:10 CDT 2011
